### ISSUE TITLE
DTAP controller: no communication with the switch

### ISSUE THREAD

**Issue by linuxstudio** [clustervision/trinity#294]
[root@controller ~]# cat /trinity/version 
    * dtap
    d32961a

This controller was installed from 'tmaster' with the dtap branch on 8 june 2015.

As it is now, the controller cannot communicate with the (Nortel) switch that has address 192.168.192.168.

Investigating the issue appears that the controller, even if it has the interface em3 correctly cabled to the management "green" vlan of the switch, doesn't have its IP address assigned:

    [root@controller ~]# ip a show em3 | grep -w 'inet'
    [root@controller ~]#

Also, the `networks` xCAT tables on this controller (dtap branch) is not correctly configured with an entry mentioning the switch network. The `hosts` xCAT table however has a correct entry for the object named "switch", with an address 192.168.192.168. Consider however that only having an entry in the  `hosts` table is not enough if there is no corresponding network defined for that address.

To fix it manually I did the following:

1) after installing the controller, assign an IP address to em3 (already properly cabled to port 4 of the Nortel switch):

    [root@controller ~]# ip a show em3 | grep -w 'inet'
        inet 192.168.192.4/24 brd 192.168.192.255 scope global em3

2) Adapt the relevant xCAT tables that refer to the "switch" or its network address:

    [root@controller ~]# tabedit networks
    ....
    "switch_net","192.168.192.0","255.255.255.0","em3",,,,,,,,,,,,,,,

(had to edit this table appending the above line)

    [root@controller ~]# tabdump hosts | grep switch
    "switch","192.168.192.168","switch",,,
    
(this table seems to be already correct)

3) Recreate /etc/hosts and update local DNS database:

[root@controller ~]# makehosts
[root@controller ~]# makedns -n

4) The switch can now be pinged and resolved from the local DNS (Test procedure 1.1.11)

In short, the things that are required, at least for the dtap branch, are:

- add "switch_net" in the `networks` xCAT table
- configure the em3 interface of the controller with an an address in the "switch_net"

**Comment by hansthen** [comment]
The controllers are supposed to move to a customer site where they most likely will not use the switch configuration as we have. I do not want to add "specializations" to our internal setup that will be exposed in customers sites. We should add the extra ip address to the controllers we install in our dtap environment manually (and preferably non-persistent).

**Comment by hansthen** [comment]
Assigning to Themis for inclusion into the test procedures.
